config/default.yml

```diff
@@ -51,7 +51,7 @@ Rails/ActionFilter:
 
 Rails/ActionOrder:
   Description: 'Enforce consistent ordering of controller actions.'
-  Enabled: false
+  Enabled: pending
```

Comment:
Have you had a chance to run the cop against `real-world-rails`?

Reply:
Yes, (man those repos took a long time to clone) I got a bunch of proposed changes ala.
```
apps/24pullrequests/app/controllers/contributions_controller.rb:24:3: C: [Correctable] Rails/ActionOrder: Controller actions should be sorted in the following order: index, show, new, edit, create, update, destroy. Action show should appear before create.
  def show ...
  ^^^^^^^^
```
but I can't manually inspect them all. I didn't find anything out of order though

Reply:
Is there an official way check against real-world-rails? It seems like something that would be best done in a github action - my macbook was not happy cloning all that stuff

Reply:
> an official way check against real-world-rails

Not that I know of.
The process is also prone to choking on local project's configs that are for earlier RuboCop versions, so you have to find and remove all `.rubocop*.yml`. You've probably already noticed that.

Reply:
It seemed to work fine by specifying a `rubocop.yml` with the `-c` option, and only running this new cop using the `--only` option